healthcheck/healthcheck.go

```diff
@@ -130,30 +130,36 @@ func (hc *HealthCheck) execute(ctx context.Context, backend *BackendConfig) {
 func (hc *HealthCheck) checkBackend(backend *BackendConfig) {
 	enabledURLs := backend.LB.Servers()
 	var newDisabledURLs []*url.URL
-	for _, url := range backend.disabledURLs {
```

Comment:
Is there a particular reason to make the variable name a single letter instead of a noun?

Reply:
because `url` is also a package name

Reply:
Is `s/u/aDisabledUrl/` a possibility (or something meaningful)?

Reply:
Has been changed into `disableURL`

Reply:
:+1: